docs/terminology.md

```diff
@@ -0,0 +1,143 @@
+the "you are logged in as..." button the upper right corner that lets
+you pick which nym you want to be.
+
+There are two related use cases:
+
+*Pseudonymity:* Consider a discussion forum web app.  In some
+contributions, a user may want to use her real name to leverage or
+increase her reputation in the community.  In others, possibly even
+under the same discussion, she may want to remain anonymous.  This can
+be achieved by using two nyms: one identical to the user's real name,
+and one generated at random.
+
+*The Moderator who also wants to engage in a debate:* If the same user
+enters a channel where she acts as moderator, she wants to have
+different controls in the UI from ordinary users (e.g., she doesn't
+want to be able to accidentally write a comment, but she needs to be
+able to delete comments of others).
+
+This can be achieved by registering several nyms with the same web
+app, and having them assigned different roles (see below).
+
+FIXME: not sure if nyms are the right tool to achieve this moderator /
+contributor double-role distinctions.  perhaps roles alone can do that
+when exposed to the UI?
+
+FIXME: see also:
+https://cms.liqd.net/intranet/copy_of_projekte/adhocracy-3/grundkonzept-und-terminologie#section-15
+
+
+### Role
+
+A role is a set that contains users and other roles.
+
+
+### Permission
+
+A permission is just a bit that tells you if an action is allowed or
+not.  A permission can be assigned to a user or a role.
+
+If permission X is assigned to role Q, then all users member in role Q
+are allowed to perform action X.
+
+
+### Group
+
+Groups are very similar to roles.  In fact while they still live
+inside thentos, they are the equivalent of roles in web apps.  Where
+roles are about users, groups are about nyms.  And where a role is
+valid in a thentos installation globally, a group is local to a web
+app registered with thentos.  (If thentos only serves one web app,
+this is not much of a distinction.  But if there are many web apps
+sharing the same thentos server, each of them can manage their own
+groups.)
+
+
+### User (Adhocracy3)
+
+Like thentos user.  See
+https://cms.liqd.net/intranet/copy_of_projekte/adhocracy-3/grundkonzept-und-terminologie#section-14
+
+
+### Group (Adhocracy3)
+
+Like thentos role.  See
+https://cms.liqd.net/intranet/copy_of_projekte/adhocracy-3/grundkonzept-und-terminologie#section-16
+
+
+### Process Role (Adhocracy3)
+
+Time-aware set of adhocracy permissions.  See
+https://cms.liqd.net/intranet/copy_of_projekte/adhocracy-3/grundkonzept-und-terminologie#section-17
+
+
+### Permission (Adhocracy3)
+
+Like thentos permission.  See
+https://cms.liqd.net/intranet/copy_of_projekte/adhocracy-3/grundkonzept-und-terminologie#section-19.
+
+
+## FIXME:
+
+should we make this an alphabetical list?
+
+what about these terms?:
+
+- authorization error
+- authorization
+- identification
+- qualification (in the sense of berlin servicekonto)
```

Comment:
also missing: "context".  a context is a path into a web app (e.g., a process or a proposal).  we may want to map personas and context onto each other, which blurs the line between app data and user data.  It is necessary to maintain the mapping inside thentos, because the mapping requires the UID, which to hide from the app is one of the core ideas of personas.